preprocessors/emotion-recognition/README.md

```diff
@@ -1,8 +1,23 @@
-This preprocessor has been created in order to detect expressions of an individual person. The output generated by a preprocessor would be ```happy, neutral or sad.``` It is to be noted that AI models learn human biases reflected in the datasets, leading to incorrect predictions. For example, when a popular emotion recognition model was used on NBA players, the model detected that African-American players were twice as angry compared to their white counterparts \cite{rhue_2019}. Hence such models should be used with caution as it might lead to offensive results. 
+# Expression Recognition Preprocessor
 
+This preprocessor has been created in order to detect expressions of an individual person. The output generated by a preprocessor would be ```happy, neutral or sad.``` The code is built on [Deepface](https://github.com/serengil/deepface). There are several available models such as FaceNet, Openface, DeepFace etc. The code can be modified to use either of the models. The modifications need to be performed on `line 105` to specify which model should be used for analysis. 
 
-In order to run the API as a docker container use the following commands: 
+The code to use this module as an API can be found in emotion-recognition.py. This module is fully versionned, the versions of the libraries used can be found in `requirements.txt` and in the `Dockerfile`.
 
-```docker build -t <image-name>```
 
-```docker run --publish <port>:5000 <image-name>```
+It is to be noted that AI models learn human biases reflected in the datasets, leading to incorrect predictions. For example, when a popular emotion recognition model was used on NBA players, the model detected that African-American players were twice as angry compared to their white counterparts \cite{rhue_2019}. Hence such models should be used with caution as it might lead to offensive results. 
```

Comment:
If you put in a reference, it needs to be in a format that someone can actually access. Please cite appropriately, and include the full reference at the bottom of the README